src/sql/base/browser/ui/panel/panel.ts

```diff
@@ -223,6 +247,22 @@ export class TabbedPanel extends Disposable {
 		if (tab.tab.view.onShow) {
 			tab.tab.view.onShow();
 		}
+		if (tab.firstElement === null || tab.firstElement === undefined) {
+			tab.firstElement = this.getFirstFocusableElement(tab.body);
```

Comment:
as before. we shouldn't be trying to define focus for tabs, they should be defining focus themselves.

Reply:
update : tried as discussed.. the tab works fine.. shift tab from element --> header still need the tracking of element